lms/static/js/spec/commerce/receipt_spec.js

```diff
@@ -0,0 +1,203 @@
+define([
+        'jquery', 'jquery.ajax-retry',
+        'js/commerce/views/receipt_view', 'common/js/spec_helpers/ajax_helpers'
+    ],
+    function ($, AjaxRetry, ReceiptView, AjaxHelpers){
```

Comment:
So it's not being used in this file, it's being used in ```lms/static/js/commerce/views/receipt_view.js```? If so it should be in that file's define array.